test/multifile/typealias/two-modules/library.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: true
```

Comment:
I think in this case the issue is clearly with serialization so the test should be moved there. But I'd like to keep the existing tests in multi-file as is. Sounds reasonable?